content/chainguard/chainguard-images/about/what-chainguard-will-build.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This doc needs a full style review but is approved interally

Reply:
ack